docs/kqueen.rst

```diff
@@ -159,6 +159,39 @@ pattern.
       - 127.0.0.0/8
       - Addresses allowed to access metrics endpoint without token
 
+
+Default User Access configuration
+_________________________________
+
+Default CRUD (Create, Read, Update, Delete) model for Kqueen user roles.
+
+Superadmin view
+~~~~~~~~~~~~~~~
+
+- ``CRUD`` all Organizations.
+- ``CRUD/Manage`` all members.
+- ``CRUD/Manage`` all member roles.
+- Full-Admin rights.
+
+Admin view
+~~~~~~~~~~
+
+- Invite/Remove members in own Organization (Email/LDAP).
+- ``CRD`` all Provisioners.
+- ``CRUD`` all Clusters.
+- Collect Prometheus metrics.
+- Full-User rights.
+
+
+User view
+~~~~~~~~~
+
+- Login.
+- ``R`` Organization members.
+- ``R`` Provisioners.
+- ``CRUD`` self Clusters.
```

Comment:
Currently user is able to manage provisioners too, should we change code first if we decided that it is a correct behavior. Also I've just remember, that policy can be set up for a particular organization,  not sure that it is tested, but now organization has policy settings

Reply:
I suggest to add CRUD provisiners here, and  change docs together with user right restriction

Reply:
IMO user can't manage provisioners at all
need to deny provisioner settings for user

Reply:
ok, so let's make it first and then push the docs

Reply:
okok, its better to deny from UI part?

Reply:
I believe it is set up in policy json in config (in api) section and should be real quick